dask/array/tests/test_optimization.py

```diff
@@ -350,7 +350,7 @@ def test_hard_fuse_slice_cases(getter):
 
 def test_dont_fuse_numpy_arrays():
     x = np.ones(10)
-    for chunks in [(5,), (10,)]:
+    for _ in [(5,), (10,)]:
```

Comment:
I think this one might actually be a bug. I suspect we want to pass `chunks=chunks` in the `da.from_array` call below...Let me look into this a bit

Reply:
I'm noticing there was never a final answer for this one, just making sure we didn't miss it before merging, @jrbourbeau can you confirm this is not a bug?